html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form action="index.html">
+    <label for="email">Email</label>
+    <div>
+      <input type="text" name="email" id="email">
+    </div>
+    <label for="cool">Why is coding cool?</label>
+    <div>
+      <select name="cool" id="cool">
+        <option value="option">Select an option</option>
+        <option value="intersting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <label for="language">Favorite Language (choose one)</label>
```

Comment:
This isn't labelling a specific input, it is just a label for the section, you can make this something like a `p` element.